client/stylesheets/_toCat.scss

```diff
@@ -15,10 +15,16 @@ GLOBAL | GLOBAL DIVS | HEADER | FOOTER **/
     background-color: yellow;
 }
 
+.tooltip > .tooltip-inner {
+    background-color: $color-hashtag !important;
+    height: 30% !important;
+    padding: 2em !important;
+}
+
 .box-title {
     background: $color-spacered !important;
     font-family: inherit;
-    color: #FFFFFF;
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces